tests/integration_tests_control_files/taylor_green_vortex_integration/CMakeLists.txt

```diff
@@ -28,10 +28,16 @@ add_test(
   WORKING_DIRECTORY ${TEST_OUTPUT_DIR}
 )
 # ----------------------------------------
-configure_file(viscous_taylor_green_vortex_restart_from_parameter_file_check.prm viscous_taylor_green_vortex_restart_from_parameter_file_check.prm COPYONLY)
+configure_file(viscous_taylor_green_vortex_restart_from_parameter_file_check.sh
+viscous_taylor_green_vortex_restart_from_parameter_file_check.sh COPYONLY)
 add_test(
   NAME MPI_VISCOUS_TAYLOR_GREEN_VORTEX_RESTART_FROM_PARAMETER_FILE_CHECK
-  COMMAND mpirun -np ${MPIMAX} ${EXECUTABLE_OUTPUT_PATH}/PHiLiP_3D -i ${CMAKE_CURRENT_BINARY_DIR}/viscous_taylor_green_vortex_restart_from_parameter_file_check.prm
+  COMMAND bash -c 
+  "./viscous_taylor_green_vortex_restart_from_parameter_file_check.sh ${EXECUTABLE_OUTPUT_PATH}
```

Comment:
Just got a thought. Does this trick work while running ctest on narval?

Reply:
Yes, it's done the same way as: https://github.com/dougshidong/PHiLiP/blob/0056e05976a85ba4ff9d623cac81d4be6c3719f6/tests/integration_tests_control_files/reduced_order/CMakeLists.txt#L9-L19